pyiron_base/master/generic.py

```diff
@@ -277,23 +277,10 @@ def _after_generic_copy_to(self, original, new_database_entry, reloaded):
                 if new_database_entry and child.master_id:
                     new_child.master_id = self.job_id
 
-    def update_master(self):
-        """
-        After a job is finished it checks whether it is linked to any metajob - meaning the master ID is pointing to
-        this jobs job ID. If this is the case and the master job is in status suspended - the child wakes up the master
-        job, sets the status to refresh and execute run on the master job. During the execution the master job is set to
-        status refresh. If another child calls update_master, while the master is in refresh the status of the master is
-        set to busy and if the master is in status busy at the end of the update_master process another update is
-        triggered.
-        """
-        master_id = self.master_id
-        project = self.project
-        self._logger.info("update master: {} {} {}".format(master_id, self.get_job_id(), self.server.run_mode))
-        if master_id is not None:
-            self._reload_update_master(
-                project=project,
-                master_id=master_id
-            )
+    def update_master(self, force_update=True):
+        super().update_master(force_update=force_update)
+
+    update_master.__doc__ = GenericJob.update_master.__doc__
```

Comment:
```suggestion
```

If it's a bare super call you can even just directly inherit the method.

Sorry for sticking my nose in, I was just jazzed to see a branch called 'cleanup' 😍

Reply:
XD

yeah the problem is `force_update` which is `True` in `GenericMaster` and `False` in `GenericJob` (parent class).

Is it anyway something to apologise for if someone starts reviewing without being asked to do so?

Reply:
> yeah the problem is force_update which is True in GenericMaster and False in GenericJob (parent class).

Ah, gotcha! Missed that subtlety. Shouldn't that mean the docstring can't be copy-pasted then?

> Is it anyway something to apologise for if someone starts reviewing without being asked to do so?

No, just being Canadian 😆 🇨🇦

Reply:
> > yeah the problem is force_update which is True in GenericMaster and False in GenericJob (parent class).
> 
> Ah, gotcha! Missed that subtlety. Shouldn't that mean the docstring can't be copy-pasted then?

Ah good point. I'll change that.

> 
> > Is it anyway something to apologise for if someone starts reviewing without being asked to do so?
> 
> No, just being Canadian 😆 🇨🇦

ah good now I know in which country I'm gonna be socially dead XD